### PR TITLE
ADCM-1569 fix allure.steps in test_adcm_check_plugin.py

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: tests/library/steps.py [hunk 1]
```diff
@@ -165,7 +154,14 @@ def delete_all_data(client):
...
-                                                    "service_id": service['id'],
-                                                    "component_id": component['id']}])
+    hc = [{"host_id": host['id'],
+           "service_id": service['id'],
+           "component_id": component['id']}]
+    return client.cluster.hostcomponent.create(cluster_id=cluster['id'], hc=hc)
+
+
+@allure.step('Check if action {action_name} state is {state_expected}')
+def check_state(action_name, state_current, state_expected):
```
Comment (dgusakov) [inline tests/library/steps.py:164]:
```suggestion
def check_action_state(action_name, state_current, state_expected):
```
Or make step title more general by removing "action" word

Comment (dgusakov) [reply tests/library/steps.py:164]: + add typing to args

File: tests/library/steps.py [hunk 2]
```diff
@@ -25,6 +25,7 @@
 )
 
 
+@allure.step('Pack bundle')
```
Comment (dgusakov) [inline tests/library/steps.py:28]:
```suggestion
@allure.step('Pack bundle form {bundledir}')
```

**Review by dgusakov — CHANGES_REQUESTED** [review]

File: tests/library/steps.py (hunk 1 above)
Comment (annademina) [reply tests/library/steps.py]: done

File: tests/functional/test_adcm_check_plugin.py [hunk 3]
```diff
@@ -4,159 +4,220 @@
...
         logs = job.log_list()
-        assert len(logs) == 2, "Logs count not equal 2, current log count {}".format(len(logs))
+        current_len = len(logs)
+        assert current_len == params['logs_amount'], \
+            f'Logs count not equal {params["logs_amount"]}, ' \
+            f'current log count {current_len}'
 
 
-@pytest.mark.parametrize("name, group_msg, task_msg, group_result, task_result", ALL_FIELDS_DATA)
+@pytest.mark.parametrize("name, group_msg, task_msg, "
```
Comment (dgusakov) [inline tests/functional/test_adcm_check_plugin.py:47]: Make it list instead of str with commas

**Review by skhomuti — APPROVED** [review]

File: tests/functional/test_adcm_check_plugin.py (hunk 3 above)
Comment (annademina) [reply tests/functional/test_adcm_check_plugin.py]: done

**Review by dgusakov — APPROVED** [review]

**Pull request merged**